storage_controller/src/http.rs

```diff
@@ -1188,4 +1238,11 @@ pub fn make_router(
                 RequestName("v1_tenant_passthrough"),
             )
         })
+        .get("/v1/safekeeper/:id", |r| {
```

Comment:
I think these endpoints should be `/control/v1/safekeeper/:id`:
* to match the node endpoints
* to respect the scheme where `v1` endpoint act as a virtual pageserver
* to keep the cplane proxy working

Reply:
Technically the URL can be whatever, so adding the `/control` prefix sounds fine.

> to respect the scheme where v1 endpoint act as a virtual pageserver

What do you mean by this?

Reply:
This comment explains it quite well (`storage_controller/src/http.rs`)
```
        // Tenant operations
        // The ^/v1/ endpoints act as a "Virtual Pageserver", enabling shard-naive clients to call into
        // this service to manage tenants that actually consist of many tenant shards, as if they are a single entity.
```

Reply:
raah, oki found it. for me that comment seems out of place and out of date because we only forward very specific set of requests. I'll add the control prefix and move it "above" that comment.

Reply:
[ed6365b](https://github.com/neondatabase/neon/pull/8879/commits/ed6365bcfeed634a6acf61b26aae53be0c0b398b)